EndToEndExample/actions/train/requirements.txt

```diff
@@ -0,0 +1,6 @@
+pandas
+numpy
+cortex-python==6.0.2a4
```

Comment:
Update this to use latest stable version 6.0.3

Reply:
I've been using this alpha release specifically to use SkillClient